src/pages/postgraphile/computed-columns.md

```diff
@@ -81,13 +83,13 @@ returns setof my_schema.users as $$
 $$ language sql stable;
 ```
 
-You can also expose additional arguments via your computed column function, and these will be exposed via GraphQL:
+You can also define parameters to your fields via your computed column function, and these will be exposed in the GraphQL schema as well:
```

Comment:
This doesn't read quite right to me; how about:

```suggestion
You can also define additional parameters in your computed column function, and these will be added to the GraphQL schema as well:
```

Reply:
It depends on what the intent and expected emphasis is. From my perspective, readers want to use computed columns to add an additional custom field in the GraphQL schema. The goal is to extend the GraphQL schema and its functionality.

You way reverts the emphasis to explaining the functionality of computed columns, while mine emphasises the desired goal.

Reply:
The sentence is great. Perhaps this addition helps to make it sufficiently explicit? 

`You can add parameters to your computed column field by defining additional parameters in your PostgreSQL function. Any parameters in addition to the required table type will then be exposed in the GraphQL schema:`

If you don't think it necessary, let's just go with your revision.

Reply:
Both are fine; go with whichever you prefer. The latter seems a little redundant to me but you're definitely in a better place to judge if it adds clarity that I am.